extensions/Worker.Extensions.CosmosDB/src/Constants.cs

```diff
@@ -0,0 +1,14 @@
+
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    internal static class Constants
+    {
+        internal const string ExtensionName = "CosmosDB";
+        internal const string ConfigurationSectionName = "AzureWebJobs";
+        internal const string ConnectionStringsSectionName = "ConnectionStrings";
```

Comment:
I think `IConfiguration` has an extension method for this: [IConfiguration.GetConnectionString(string)](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.configurationextensions.getconnectionstring?view=dotnet-plat-ext-7.0#microsoft-extensions-configuration-configurationextensions-getconnectionstring(microsoft-extensions-configuration-iconfiguration-system-string))